share/examples/tests/tests/atf/Makefile

```diff
@@ -46,7 +46,3 @@ file1:
 	@echo "File 1" > ${.TARGET}
 
 .include <bsd.test.mk>
-
-.if ${COMPILER_TYPE} == "clang" && ${COMPILER_VERSION} >= 180000
-CWARNFLAGS.printf_test.c+=	-Wno-format-truncation
```

Comment:
your code doesn't fix the printf truncation.
the examples that trigger printf truncation warnings do so because they test for 69c8025a498450b6bc4cc9e97ba779ed1c2e7f4b